packages/ui-kit/src/components/rss-feed-table.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import css from '@emotion/css';
+import moment from 'moment';
+import Link from './link';
+import ContentNotifications from './content-notifications';
+import { colors, tokens, dimensions } from '../design-system';
+
+const Table = styled.table`
+  width: 100%;
+  border-radius: ${tokens.borderRadiusForRssFeedTable};
+  box-shadow: ${tokens.shadowForRssFeedTable};
+
+  thead {
+    th {
+      :first-of-type {
+        border-radius: ${tokens.borderRadiusForRssFeedTable} 0 0 0;
+      }
+      :last-of-type {
+        border-radius: 0 ${tokens.borderRadiusForRssFeedTable} 0 0;
+      }
+      background-color: ${colors.light.surfaceSecondary2};
+      padding: ${dimensions.spacings.s};
+    }
+  }
+
+  tbody {
+    tr {
+      td {
+        padding: ${dimensions.spacings.m};
+
+        :first-of-type {
+          color: ${colors.light.linkHover};
+        }
+      }
+
+      :not(:first-of-type) {
+        border-top: 1px solid ${colors.light.borderPrimary};
+      }
+    }
+  }
+`;
+
+const DateWrapper = styled.span`
+  white-space: nowrap;
+`;
+
+const RssFeedTable = (props) => {
+  if (!props.data) {
+    const message = 'Must pass data to RssFeedTable component.';
+    if (process.env.NODE_ENV !== 'production') {
+      return <ContentNotifications.Error>{message}</ContentNotifications.Error>;
+    }
+
+    throw new Error(message);
+  }
+
+  return (
+    <div>
+      <Table>
+        <thead>
+          <tr>
+            <th>Release Date</th>
+            <th>Area</th>
```

Comment:
FYI: I wasn't sure about the naming here...I asked the docs team and they were for "Area". But it's still in discussion. Let me know what you think about it